dialer.go

```diff
@@ -27,15 +34,36 @@ type Dialer struct {
 	configPB   []byte
 }
 
-func NewDialer() *Dialer {
+func NewDialer(is_ios bool) *Dialer {
```

Comment:
This can later be changed to a switch for users to choose which mode they want to use (mainly for Android rn, iOS need some more dig for the compiler mode to work).